kots-exporter/kots-exporter.sh

```diff
@@ -346,8 +347,26 @@ rm_kots_annot_label_resources(){
     echo "Kots removal logs are available - $path/logs/$kotsCleanupLogFile"
 
     echo "Deleting job inject-bottoken-xxxx"
-    kubectl -n $namespace delete "$(kubectl -n $namespace get jobs -o name | grep inject)" \
-    || echo "Manually run the command: kubectl -n $namespace delete $(kubectl -n $namespace get jobs -o name | grep inject)"
+    if [ "$(kubectl -n $namespace get jobs -o name | grep inject)" != "" ]; then
+    kubectl -n $namespace delete "$(kubectl -n $namespace get jobs -o name | grep inject)"
+    fi
+}
+
+create_postgres_migration(){
+    if "$POSTGRES_INTERNAL"; then
+    echo ""
+    echo "############ CREATING POSTGRES MIGRATION SCRIPT ############"
+
+    echo "Upgrading to server CircleCI Server 4.0 includes upgrading the Postgres chart"
+    echo "Before upgrading, we need to prepare your postgres instance."
+    echo "Migration command will be saved in $path/output/postgres-migration.sh"
+
+    echo "export POSTGRESQL_PASSWORD='$(kubectl get secret --namespace $namespace postgresql -o jsonpath='{.data.postgresql-password}' | base64 --decode)'" > "$path"/output/postgres-migration.sh
+    # shellcheck disable=SC2129
+    echo "export POSTGRESQL_PVC='$(kubectl get pvc --namespace $namespace -l app.kubernetes.io/instance=circleci-server,role=primary -o jsonpath='{.items[0].metadata.name}')'" >> "$path"/output/postgres-migration.sh
+    echo "kubectl delete statefulsets.apps postgresql --namespace $namespace --cascade=orphan" >> "$path"/output/postgres-migration.sh
+    echo "kubectl delete secret postgresql --namespace $namespace" >> "$path"/output/postgres-migration.sh
```

Comment:
These can be run as part of the script, they customer does not need to run them

Reply:
To keep it together all the postgres migration command

Reply:
I don't know what that means.  Are there other commands the customer is asked to run after the script?